core/arch/arm/plat-imx/main.c

```diff
@@ -177,6 +176,22 @@ vaddr_t pl310_base(void)
 	return PL310_BASE;
 }
 
+void arm_cl2_config(vaddr_t pl310 __unused)
```

Comment:
this `arm_cl2_config` is only used before mmu enabled, the physical addr is fine.